.buildkite/pull-requests.json

```diff
@@ -19,3 +19,20 @@
    ]
 }
 
+{
+   "jobs": [
+     {
+        "enabled": true,
+        "pipelineSlug": "package-spec",
+        "allow_org_users": true,
+        "allowed_repo_permissions": ["admin", "write"],
+        "allowed_list": [ ],
+        "set_commit_status": true,
+        "build_on_commit": true,
+        "build_on_comment": true,
+        "trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:build|test)\\W+(?:this|it))|^/test$",
+        "always_trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:build|test)\\W+(?:this|it))|^/test$",
+        "fail_on_not_mergeable": true,
+     }
```

Comment:
this entry needs to be inside the previous JSON array:

```
{
   "jobs": [
     {
        "enabled": true,
        "pipelineSlug": "package-spec-test-with-integrations",
        "allow_org_users": true,
        "allowed_repo_permissions": ["admin", "write"],
        "allowed_list": [ ],
        "set_commit_status": false,
        "build_on_commit": false,
        "build_on_comment": true,
        "trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:test)\\W+(?:integrations))$",
        "always_trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:test)\\W+(?:integrations))$",
        "skip_ci_labels": [ ],
        "skip_target_branches": [ ],
        "skip_ci_on_only_changed": [ ],
        "always_require_ci_on_changed": [ ]
     },
     {
        "enabled": true,
        "pipelineSlug": "package-spec",
        "allow_org_users": true,
        "allowed_repo_permissions": ["admin", "write"],
        "allowed_list": [ ],
        "set_commit_status": true,
        "build_on_commit": true,
        "build_on_comment": true,
        "trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:build|test)\\W+(?:this|it))|^/test$",
        "always_trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:build|test)\\W+(?:this|it))|^/test$",
        "fail_on_not_mergeable": true,
     }
   ]
}
```

Reply:
fixed